pkg/loadbalancer/proxy.go

```diff
@@ -90,6 +90,13 @@ static_resources:
     connect_timeout: 0.25s
     type: STATIC
     lb_policy: ROUND_ROBIN
+    health_checks:
+      - timeout: 1s
+        interval: 2s
+        unhealthy_threshold: 3
+        healthy_threshold: 1
```

Comment:
again would be good to explain why these particular numbers

Reply:
copy paste

Reply:
maybe at least 

`// FIXME: figure out what these numbers mean and if we should change them or make them configurable`

Reply:
(though fwiw the "real" cloud providers have tons of random hardcoded constants in their health check configs too)

Reply:
this is explained here https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/health_check.proto.html

I finally understood the API docs, is just that all of them are defined in proto and they render this way on the docs and at least to me was hard to find the values and the meaning

Reply:
I prefer to not expose low level options like this to the users until we have a super stable version that works in CI and we have some strong use cases